app/controllers/users_controller.rb

```diff
@@ -34,7 +34,7 @@ def new_with_invitation_token
   private
     
   def user_params
-    params.require(:user).permit(:email, :password, :full_name)
+    params.require(:user).permit(:email, :password, :full_name, :admin)
```

Comment:
This action was added for the db:seed, adding a admin user.
I should remove it.